ReleaseNotes-2.9.2

```diff
@@ -29,3 +29,5 @@ Bugfixes
 
 Frontend:
   * Fixed issue in live build log that caused parts of the log being duplicated
+  * Upgrading from 2.8 to 2.9 caused remote repositories with same name to get deleted
+    - If you already upgraded and have an interconnect configured, a consistence check with 'rake check_project <project>' might be necessary 
```

Comment:
I would a pointer here, telling a user what to do if the consistency checker find inconsistencies. Be it some concrete task or just that the user should reach out to us via $xy to get help